myapp/src/App.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <div>
+    <p>
+      {{msg}}
```

Comment:
`{}`の両内側にはスペースを空けましょう。
```suggestion
      {{ msg }}
```